content/overview/quick-start.md

```diff
@@ -35,7 +35,7 @@ pip install farm-haystack
 
 {{< tab tabName="Basic"  >}}
 
-This command installs everything you need for basic Pipelines that use an InMemoryDocumentStore, as well as all necessary dependencies for model inference on local machine, including torch.
+This command installs everything you need for basic Pipelines that use an InMemoryDocumentStore, and necessary dependencies for model inference on a local machine, including torch.
```

Comment:
Same here: I'd tell them what they can/cannot do with this installation. Can they create a working pipeline on their machine? what are the limitations? etc